### PR TITLE
Kaushik kc/notification list overflows the viewport in mobile

### LINKED ISSUE #2446: Notification list overflows the viewport in mobile
## Bug description

Tested with the Chrome mobile emulator, using "iPhone 12 pro" with 390 x 844 resolutioni

## Environment
 - Browser: Chrome
 - Wallet: any
 - Chain: any
 - Env: Prod 1.17

## Steps to reproduce
 1. Open notifications in the mobile viewport (it can be empty, no need to have notifications in it)

## Obtained result
image

### PR DESCRIPTION
**Pull request by KaushikKC**
## What it solves

Resolves #2446

## How this PR fixes it
I have used the Css media query to adjust the width of the notification bar in the mobile screen

## How to test it
open the site in the developer mode and view in different mobile width.

## Screenshots
Screenshot from 2023-08-29 18-51-37

## Checklist
- [x] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

**Comment by katspaugh** [comment]
Thanks for the PR! Could you post a screenshot with actual notifications in the popup?

File: src/components/notification-center/NotificationCenter/styles.module.css [hunk 1]
```diff
@@ -13,6 +13,13 @@
   border: 1px solid var(--color-border-light);
 }
 
+@media only screen and (max-width: 470px) {
```
Comment (usame-algan) [inline src/components/notification-center/NotificationCenter/styles.module.css:16]: We are using MUI in this project and try to adhere to the default breakpoints as much as possible. In this case I would suggest to use the max-sm breakpoint which is 599.95px.

**Comment by KaushikKC** [comment]
Here is the screenshot of the actual notifications in the popup

Screenshot from 2023-09-01 09-36-13

I have also added the MUI breakpoints to solve this.

I have read the CLA Document and I hereby sign the CLA

**Comment by KaushikKC** [comment]
recheck

**Comment by KaushikKC** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by KaushikKC** [comment]
recheck

**Comment by usame-algan** [comment]
@KaushikKC can you check the diff? It says 1138 files changed. Looks like an OS formatting issue.

**Comment by katspaugh** [comment]
It's not a formatting issue, the dev branch was rebased.
@KaushikKC could you rebase your commits on the latest dev? Thanks!

**Comment by KaushikKC** [comment]
> It's not a formatting issue, the dev branch was rebased. @KaushikKC could you rebase your commits on the latest dev? Thanks!

Yeah done

**Comment by katspaugh** [comment]
There’s still some git issue. Looks like the permissions of each file were changed.

**Comment by katspaugh** [comment]
I've merged these changes here: https://github.com/safe-global/safe-wallet-web/commit/e200a0ec60897897cee9d52c8c057f1213d4ee58
Thanks for the contribution!

**Pull request closed without merging**